extensions/VisualEditor/modules/ve-mw/ui/dialogs/ve.ui.MWSaveDialog.js

```diff
@@ -460,11 +455,13 @@ ve.ui.MWSaveDialog.prototype.initialize = function () {
 ve.ui.MWSaveDialog.prototype.getSetupProcess = function ( data ) {
 	return ve.ui.MWSaveDialog.super.prototype.getSetupProcess.call( this, data )
 		.next( function () {
+			var pageExists = mw.config.get( 'wgArticleId', 0 ) !== 0;
+
 			// Old messages should not persist
 			this.clearAllMessages();
 			this.swapPanel( 'save' );
 			// Update save button label
-			this.actions.forEach( { actions: 'save' }, function ( action ) {
+			this.getActions().forEach( { actions: 'save' }, function ( action ) {
 				action.setLabel(
 					ve.msg(
 						// TODO: Actually populate this.resotring with information, right now it is
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)